apps/src/blockly/addons/cdoFieldDropdown.js

```diff
@@ -1,31 +1,8 @@
 import GoogleBlockly from 'blockly/core';
-import i18n from '@cdo/locale';
-
-const EMPTY_OPTION = '???';
```

Comment:
I moved `EMPTY_OPTION` and `NO_OPTIONS_MESSAGE` constants to `blockly/constants.js`.